### PR TITLE
ability to provide an app level component action handler

### PR DESCRIPTION
**Pull request by jedireza**
Let's review and discuss the approach. When things solidify I'll add docs.

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: lib/Fluxible.js [hunk 1]
```diff
@@ -69,6 +70,18 @@ Fluxible.prototype.createContext = function createContext(options) {
 };
 
 /**
+ * The default component action handler
+ * @method defaultActionHandler
+ * @param {Error} [err]
```
Comment (redonkulus) [inline lib/Fluxible.js]: @private

File: lib/Fluxible.js [hunk 2]
```diff
@@ -69,6 +70,18 @@ Fluxible.prototype.createContext = function createContext(options) {
...
+ */
+Fluxible.prototype.defaultActionHandler = function defaultActionHandler(err) {
```
Comment (redonkulus) [inline lib/Fluxible.js]: Underscore

Comment (redonkulus) [inline lib/Fluxible.js]: defaultComponentActionHandler

File: docs/fluxible-context.md [hunk 3]
```diff
@@ -53,6 +53,8 @@ This action context object is used every time an `executeAction` method is calle
 
 Generates a context interface providing access to only the functions that should be called from components. By default: `executeAction`, `getStore`. `executeAction` does not allow passing a callback from components so that it enforces actions to be send and forget.
 
+Note: You may provide an app level `componentActionHandler` function when instantiating Fluxible. This allows you to catch errors (at a high level) spawning from components firing actions.
```
Comment (jedireza) [inline docs/fluxible-context.md]: Should we mention intended use cases (or something about 404/500 errors)?

Comment (redonkulus) [inline docs/fluxible-context.md]: We could even just link to the fluxible.io docs once they are up for use cases. Adding associated docs to the Controller View guide.

**Comment by redonkulus** [comment]
Looks ok to me. Have @mridgway take a look too.

**Comment by mridgway** [comment]
Looks good to me.

**Pull request merged**